eth/downloader/stagedsync_stages.go

```diff
@@ -29,7 +29,8 @@ type SyncStage byte
 
 const (
 	Headers   SyncStage = iota // Headers are downloaded, their Proof-Of-Work validity and chaining is verified
-	Bodies                     // Block bodies are downloaded, TxHash and UncleHash are getting verified, "From" recovered from signatures
+	Bodies                     // Block bodies are downloaded, TxHash and UncleHash are getting verified
```

Comment:
maybe we will have something like `Bodies []byte("bodies")` instead of them being a number?
with numbers we are making it harder to insert stages mid-process.